### PR TITLE
workflows: add dependabot recreate pull request workflow

### PR DESCRIPTION
**Pull request by jelly**

### CODE REVIEW

File: .github/workflows/rebase-dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: rebase-dependabot-pr
+on:
+  # push:
+  #   branches: [ main ]
```
Comment (martinpitt) [inline .github/workflows/rebase-dependabot.yml:4]: Ah, you want to test first and enable in a second step? That's wise -- the new workflow has to be on main first to be able to trigger it on the UI.

File: .github/workflows/rebase-dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+      contents: write
+    steps:
+      - name: Set up secrets
+        run: echo '${{ secrets.GITHUB_TOKEN }}' > ~/.config/github-token
+
+      - name: Checkout bots repo
+        uses: actions/checkout@v4
+        with:
+          repository: cockpit-project/bots
+          fetch-depth: 0
```
Comment (martinpitt) [inline .github/workflows/rebase-dependabot.yml:21]: Default of 1 should be fine here, no? bots doesn't have any tags or so

**Review by martinpitt — DISMISSED** [review]
Thanks! Let's land and give it a test. We can clean up the fetch-depth once we get a successful manual run and enable the `push` trigger.

**Review by martinpitt — DISMISSED** [review]
I actually *liked* commenting out the `push` trigger at first, but sure!

**Comment by jelly** [comment]
> I actually _liked_ commenting out the `push` trigger at first, but sure!

Let's do that!

**Review by martinpitt — APPROVED** [review]
Yes! No! Yes! :grin: 

Thanks!

**Pull request merged**